src/pod/pod_str.rs

```diff
@@ -96,31 +94,40 @@ mod tests {
     #[test]
     fn test_from() {
         let str = PodStr::<10>::from("str");
-        assert_eq!(&str, "str");
+        assert_eq!(str.as_str().unwrap(), "str");
+    }
+
+    #[test]
+    fn test_invalid_bytes() {
+        // Invalid utf-8 bytes. The fourth byte has to be 10xxxxxx.
+        let invalid_bits: [u8; 4] = [0b1111_0000, 0b1100_0000, 0b1100_0000, 1];
+        let mut str = PodStr::<10>::default();
+        str.copy_from_slice(&invalid_bits);
+        assert!(str.as_str().is_err());
     }
 
     #[test]
     fn test_copy_from_slice() {
         let mut str = PodStr::<10>::from("empty");
-        assert_eq!(&str, "empty");
+        assert_eq!(str.as_str().unwrap(), "empty");
```

Comment:
For consistency, should we use `?` and return a `Result`, like the `*PrefixStr` tests?